src/gazebo_mavlink_interface.cpp

```diff
@@ -593,10 +582,23 @@ void GazeboMavlinkInterface::SendSensorMessages()
 #endif
   ignition::math::Vector3d pos_n = q_ng.RotateVector(pos_g);
 
-  float declination = get_mag_declination(groundtruth_lat_rad, groundtruth_lon_rad);
+  // Magnetic declination and inclination (radians)
+  float declination = get_mag_declination(groundtruth_lat_rad * 180 / M_PI, groundtruth_lon_rad * 180 / M_PI) * M_PI / 180;
+  float inclination = get_mag_inclination(groundtruth_lat_rad * 180 / M_PI, groundtruth_lon_rad * 180 / M_PI) * M_PI / 180;
+
+  // Magnetic strength (10^5xnanoTesla)
+  float strength = 0.01f * get_mag_strength(groundtruth_lat_rad * 180 / M_PI, groundtruth_lon_rad * 180 / M_PI);
```

Comment:
Nitpick: my taste is to add the unit  if possible, e.g. `strength_ga`.

Reply:
Done